x-pack/plugins/enterprise_search/KEA.md

```diff
@@ -0,0 +1,178 @@
+# Kea State Management
+Enterprise Search uses [Kea.js](https://github.com/keajs/kea) to manage our React/Redux state for us. Kea state is handled in our `*Logic` files and exposes [values](https://kea.js.org/docs/guide/concepts#values) and [actions](https://kea.js.org/docs/guide/concepts#actions) for our components to get and set state with.
```

Comment:
I think while we are still on 2.X of kea we should make sure all links to docs point to the v2 docs https://v2.keajs.org/docs/guide/concepts

Reply:
Good point!